pennylane/_qubit_device.py

```diff
@@ -111,32 +111,14 @@ def __init__(self, wires=1, shots=1000, analytic=True):
 
     @classmethod
     def capabilities(cls):
-        """Get the capabilities of the plugin.
 
-        Capabilities include:
-
-        * ``"model"`` (*str*): either ``"qubit"`` or ``"CV"``.
-
-        * ``"inverse_operations"`` (*bool*): ``True`` if the device supports
-          applying the inverse of operations. Operations which should be inverted
-          have ``operation.inverse == True``.
-
-        * ``"tensor_observables" (*bool*): ``True`` if the device supports
-          expectation values/variance/samples of :class:`~.Tensor` observables.
-
-        The qubit device class has built-in support for tensor observables. As a
-        result, devices that inherit from this class automatically
-        have the following items in their capabilities
-        dictionary:
-
-        * ``"model": "qubit"``
-        * ``"tensor_observables": True``
-
-        Returns:
-            dict[str->*]: results
-        """
-        capabilities = cls._capabilities
-        capabilities.update(model="qubit", tensor_observables=True)
+        capabilities = super().capabilities().copy()
```

Comment:
I tried outsourcing this line to the base device class in a function that can be inherited, but it creates a big mess with the ``super()`` function. So I left it here.